File: ce/outlook-app/snippets/visualbasic/CRMV8/dataimportvb/vb/importwithcreate6.vb
' <snippetimportwithcreate6>


 ' Upload the records.
 Dim importRequest As New ImportRecordsImportRequest() With {.ImportId = importId}
 Dim importResponse As ImportRecordsImportResponse =
     CType(_serviceProxy.Execute(importRequest), ImportRecordsImportResponse)
 Console.WriteLine("Waiting for ImportRecords async job to complete")

' </snippetimportwithcreate6>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/dataimport/cs/importwithcreate6.cs
// <snippetimportwithcreate6>


 // Upload the records.
 ImportRecordsImportRequest importRequest = new ImportRecordsImportRequest()
 {
     ImportId = importId
 };
 ImportRecordsImportResponse importResponse =
     (ImportRecordsImportResponse)_serviceProxy.Execute(importRequest);
 Console.WriteLine("Waiting for ImportRecords async job to complete");

// </snippetimportwithcreate6>